BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -77,38 +77,38 @@ record(dfanout, "$(P)$(R)RFFEPidSp-SP") {
 }
 
 record(ao, "$(P)$(R)RFFEPidSpAC-SP") {
-    field(DESC, "Set AC board heater temperature point")
+    field(DESC, "Set AC board heater temp. setpoint")
     field(DTYP, "stream")
     field(PINI, "YES")
     field(SCAN, "Passive")
-    field(OUT, "@bpmrffe.proto outHeaterTempAC $(PORT)")
+    field(OUT, "@bpmrffe.proto outPIDTempSpAC $(PORT)")
```

Comment:
I don't think the protocol operation has to be renamed to contain PID, but I'm not fully against it either.

Reply:
I did so to make it clearer that the temperature we are setting is the one that will be used by the PID controller. I don't know if a yet another component is involved with setpoint value, but it seems to me that PID controller is the main one impacted by it.

Reply:
I've noticed now this is related to the PV naming as well. I recall that you didn't like much the original name, which is `RFFEPidSp` (`RFFEPidSpAC` and `RFFEPidSpBD`).

Do you have a suggestion on how to better name this?

Reply:
For the PV, I'd do `RFFETemp{,AC,BD}-SP`

What you're controlling is the temperature setpoint, it's a matter of implementation that it will be controlled using a PID controller.

Reply:
It sounds good for me as well. I'll add this to issue #2, so that we don't miss it.